cost/superseded_instance_remediation/README.md

```diff
@@ -0,0 +1,35 @@
+## Superseded Instance Types Remediation Policy
+
+### What it does
+
+This Policy Template is used to automatically resize instances based on user-defined standards.  For example, the Policy uses a Disallowed Instance Type and its corresponding Superseded Instance Type.  If any Instances are found using the Disallowed type, they will be included in the corresponing Policy Incident.  Upon approval, Cloud Workflow will resize the Instance to the Superseded type. 
+
+### Prerequesites
+- The following RightScale Credentials
+  - Cloud Management - `Actor`
+  - Cloud Management - `Observer`
+
+### Functional Details
+
+- This policy supports a single old-to-new instance type mapping.  To apply this policy for multiple old-to-new instance type scenarios, the policy will need to be applied multiple times.  This will allow for flexibility in managing applied policies.
```

Comment:
applying multiple policies may be confusing to a policy user.  i'm wondering if some examples should be giving.